src/components/RangeSlider/index.js

```diff
@@ -16,13 +16,28 @@ class RangeSlider extends Component {
     }
   }
 
+  roundLabelValue = value => {
+    const {
+      incrementSize
+    } = this.props
+
+    if (!isFinite(incrementSize) || incrementSize <= 0) return value;
+    var e = 1, p = 0;
```

Comment:
We'll want to use `let` here and we elide trailing semi-colons. I'm not surprised we don't have linting setup on this repo.